SDF-to-Coords

```diff
@@ -0,0 +1,29 @@
+import os
+
+import numpy as np
+
+def SDF_to_Coords(filename, num_atom):
+    directory = r'/mnt/d/SDFs/structures/'
+    for filename in os.listdir(directory):
+        if filename.endswith(".sdf"):
+            with open(filename) as f:
+                xyz = []
+                counter = 0
+                for line in f:
+                    counter += 1
+                    if counter < 4:
+                        continue
+                      
+                    elif counter > 4 and counter <= 4+num_atom:
+                        xyz.append(line.split()[0:4])
+                        continue
+                        
+                    elif counter > 4+num_atom:
+                        pass
+                    
+            xyz = np.asarray(xyz)
+            for atom in xyz:
+                i = [3,0,1,2]
+                xyz = xyz[:,i]
+                break
+            return(xyz)
```

Comment:
This code works. As per our conversation, I'll now try to define a function where the user specifies the atoms in the functional group as an atom[ ] list, and a fnct_grp input array pops out.

Reply:
Your for loop on lines 25 to 28 isn't actually iterating. Your code will work if you take lines 26 and 28 out of the loop.

Reply:
Good point, I can see than now.